bin/fba_launch

```diff
@@ -31,6 +31,7 @@ from fiberassign.fba_launch_io import (
     mv_temp2final,
     copy_to_svn,
 )
+from fiberassign.utils import freeze_iers
```

Comment:
This should be `from desiutil.iers import freeze_iers`. If there is a specific reason for this import indirection, please describe it in detail. My personal feeling is that this indirection will only cause serious confusion.